types/places.js/index.d.ts

```diff
@@ -0,0 +1,340 @@
+// Type definitions for places.js 1.16
+// Project: https://community.algolia.com/places
+// Definitions by: Samuel Jaeschke <https://github.com/samjetski>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+import * as algoliasearch from 'algoliasearch';
+import { EventEmitter } from 'events';
+
+export as namespace places;
+
+export = places;
+
+declare function places(options: places.PlacesStaticOptions): places.PlacesAutocomplete;
+
+declare namespace places {
+    enum PlaceType {
```

Comment:
```suggestion
    const enum PlaceType {
```

`places` doesn't actually have an exported *value* named `PlaceType`, so callers would be broken if they wrote `PlaceType.City` since `PlaceType` is `undefined`.

Reply:
Ah cheers, that makes a lot of sense! The default dtslint rules appear to disallow `const enum`. I'll put an exception, but is that supposed to be so?